src/js/components/Settings/VoterPhoneVerificationEntry.jsx

```diff
@@ -650,20 +646,22 @@ class VoterPhoneVerificationEntry extends Component {
               {voterSMSPhoneNumberFromList.sms_ownership_is_verified ?
                 null : (
                   <SecondRowPhoneOrEmail>
-                    <span
-                      className="u-link-color u-cursor--pointer u-no-break"
-                      onClick={() => this.reSendSignInCodeSMS(voterSMSPhoneNumberFromList.normalized_sms_phone_number)}
-                    >
-                      Send verification again
-                    </span>
-                    {allowRemoveSMSPhoneNumber && (
-                      <TrashCan
-                        className="u-link-color u-cursor--pointer"
-                        onClick={() => this.removeVoterSMSPhoneNumber.bind(this, voterSMSPhoneNumberFromList.sms_we_vote_id)}
+                    <SecondRowPhoneOrEmailDiv>
+                      <span
+                        className="u-link-color u-cursor--pointer u-no-break"
+                        onClick={() => this.reSendSignInCodeSMS(voterSMSPhoneNumberFromList.normalized_sms_phone_number)}
                       >
-                        <Delete />
-                      </TrashCan>
-                    )}
+                        Send verification again
+                      </span>
+                      {allowRemoveSMSPhoneNumber && (
+                        <div
```

Comment:
We want to maintain the use of the named styled-div "TrashCan" for readability. It is pulled in from ../Style/pageLayoutStyles -- could you please see all other places that styled-div is used and see if any changes will affect the other places "TrashCan" is being used?